src/es/home.html

```diff
@@ -0,0 +1,121 @@
+
+      <p>Casi con toda seguridad, como resultado directo de nuestra defensa, Apple ha contratado a docenas de empleados adicionales
+para trabajar en WebKit y Safari. Este ancho de banda adicional y la presión regulatoria han contribuido directamente
+a la incorporación de funciones que se habían demorado durante mucho tiempo, como las notificaciones push y las credenciales de aplicaciones para las aplicaciones web.
+Las cifras del personal de WebKit son secretas, pero creemos que se trata de un aumento porcentual significativo.</p>
+
+      <p><strong>La competencia, o al menos la amenaza de ella, funciona.</strong></p>
+
+      <p>OWA necesita tu ayuda para seguir ejerciendo presión para restablecer la competencia y hacer realidad
+las aplicaciones web universales. Puedes marcar una verdadera diferencia para garantizar el futuro de la
+web apoyando esta labor. Dependemos de las donaciones para desafiar a los guardianes increíblemente ricos
+que tienen todos los incentivos para frenar el avance de la web.</p>
+
+      <p>Puedes<a href="/donate">donar</a> o únete a nuestra comunidad en
+        <a href="https://discord.com/invite/x53hkqrRKx">Discord</a> para involucrarte más profundamente. ¿No puedes prescindir del
+dinero ni el tiempo? Manténgate al tanto de lo que hacemos y ayude a difundir la información siguiéndonos en
+        <a href="https://twitter.com/OpenWebAdvocacy">Twitter</a> y/o <a href="https://mastodon.social/@owa">Mastodon</a>
+        .</p>
+    </div>
+
+    <div class="prom-banner">
+      <p class="illustration"><img src="/images/donate.svg" alt="" /></p>
+      <p>Si te gusta lo que hacemos y te gustaría apoyar nuestro trabajo, por favor considera
+        <a href="https://www.paypal.com/donate/?hosted_button_id=3FD5DUWT4DNBG">haciendo una donación</a> o consiguiendo que tu empresa nos apoye.</p>
+      <p>OWA es una organización sin fines de lucro registrada que lucha por la Web contra los guardianes fuertemente financiados para asegurar el futuro de la competencia de los navegadores y las aplicaciones web</p>
+
+      <div>
+          <p><a href="https://www.paypal.com/donate/?hosted_button_id=3FD5DUWT4DNBG" class="donate-button">Done ahora
+              <svg stroke="currentColor" fill="none" stroke-width="2" viewBox="0 0 24 24" stroke-linecap="round" stroke-linejoin="round" height="1em" width="1em" xmlns="http://www.w3.org/2000/svg"><circle cx="12" cy="12" r="10"></circle><polyline points="12 16 16 12 12 8"></polyline><line x1="8" y1="12" x2="16" y2="12"></line></svg>
+          </a></p>
+      </div>
+    </div>
+
+    <div class="action-button-row">
+      <a href="/donate/" class="action-button" aria-label="Lea más sobre cómo apoyar a Owa ">
+        Leer más</a>
+      <a href="/contributors/" class="action-button">Sobre nosotras</a>
+    </div>
+  </div>
+
+  <div class="contact-entry">
+    <h2>¿Como puedes ayudar?</h2>
```

Comment:
```suggestion
    <h2>¿cómo puedes ayudar?</h2>
```
lower case as the rest of the document (although I find it strange) and use an accent because it's a question